### ISSUE TITLE
Fix MICrONS Errors

### ISSUE THREAD

**Issue by JEElsner** [AllenSWDB/allenswdb.github.io#82]
These two pages throw the same error, stemming from importing `cloudvolume`. These errors are probably related/one-and-the-same. Given that it's from importing `cloudvolume`, this might be a bug in that package.
- [ ] [`anatomy/microns-em/em-skeletons.md`](https://allenswdb.github.io/anatomy/microns-em/em-skeletons.html#annotations) Execution Error
- [x] [`anatomy/microns-em/em-volume-data.md`](https://allenswdb.github.io/anatomy/microns-em/em-volume-data.html) Execution Error

## Build log warnings
```
/opt/databook/databook/anatomy/microns-em/em-skeletons.md: WARNING: Executing notebook failed: CellExecutionError [mystnb.exec]
/opt/databook/databook/anatomy/microns-em/em-skeletons.md: WARNING: Notebook exception traceback saved in: /root/capsule/results/_build/html/reports/anatomy/microns-em/em-skeletons.err.log [mystnb.exec]

/opt/databook/databook/anatomy/microns-em/em-volume-data.md: WARNING: Executing notebook failed: CellExecutionError [mystnb.exec]
/opt/databook/databook/anatomy/microns-em/em-volume-data.md: WARNING: Notebook exception traceback saved in: /root/capsule/results/_build/html/reports/anatomy/microns-em/em-volume-data.err.log [mystnb.exec]
```

## Stack trace
This one is for `em-skeletons.md`
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[2], line 1
----> 1 from meshparty import meshwork
      2 nrn = meshwork.load_meshwork('data/864691134940133219_skel.h5')
      3 nrn.anno.post_syn.df.head()

File /opt/envs/allensdk/lib/python3.8/site-packages/meshparty/meshwork/__init__.py:1
----> 1 from .meshwork import Meshwork, load_meshwork
      2 from . import algorithms

File /opt/envs/allensdk/lib/python3.8/site-packages/meshparty/meshwork/meshwork.py:9
      6     _vtk_loaded = False
      8 from ..skeleton_io import swc_node_labels
----> 9 from ..trimesh_io import Mesh
     10 from ..skeleton import Skeleton, resample
     12 import contextlib

File /opt/envs/allensdk/lib/python3.8/site-packages/meshparty/trimesh_io.py:19
     17 import logging
     18 from functools import wraps
---> 19 import cloudvolume
     20 from cloudvolume.datasource.precomputed.mesh.multilod import ShardedMultiLevelPrecomputedMeshSource
     21 from multiwrapper import multiprocessing_utils as mu

File /opt/envs/allensdk/lib/python3.8/site-packages/cloudvolume/__init__.py:71
     68 __version__ = '10.0.0'
     70 # Register plugins
---> 71 from .datasource.precomputed import register as register_precomputed
     72 from .datasource.graphene import register as register_graphene
     73 from .datasource.n5 import register as register_n5

File /opt/envs/allensdk/lib/python3.8/site-packages/cloudvolume/datasource/precomputed/__init__.py:3
      1 from typing import Optional, Union
----> 3 from .image import PrecomputedImageSource
      4 from .metadata import PrecomputedMetadata
      5 from .mesh import PrecomputedMeshSource

File /opt/envs/allensdk/lib/python3.8/site-packages/cloudvolume/datasource/precomputed/image/__init__.py:28
     26 from .... import paths
     27 from ....lib import Bbox, Vec, sip, first, BboxLikeType, toiter
---> 28 from .... import sharedmemory, chunks
     30 from ... import autocropfn, readonlyguard, ImageSourceInterface
     31 from .. import sharding

File /opt/envs/allensdk/lib/python3.8/site-packages/cloudvolume/chunks.py:456
    446     arr = decode(filedata, encoding, shape, dtype, block_size, 0)
    447     return bool(np.isin(label, arr))
    449 def transcode(
    450   image_chunks:Union[
    451     Dict[Union[str,int], bytes], # { label: binary }
    452     Sequence[Tuple[Union[str,int], bytes]] # ( (label, binary) for label, binary in ... )
    453   ], 
    454   src_encoding:str, 
    455   dest_encoding:str,
--> 456   chunk_size_fn:Callable[Union[str,int], ShapeType],
    457   dtype:Union[str,np.dtype],
    458   background_color:int = 0,
    459   progress:bool = False,
    460   in_place:bool = False,
    461   src_block_size:ShapeType = DEFAULT_CSEG_BLOCK_SIZE,
    462   dest_block_size:ShapeType = DEFAULT_CSEG_BLOCK_SIZE,
    463   compression_params:dict = {},
    464   force:bool = False,
    465   total:Optional[int] = None,
    466 ):
    467   """
    468   Convert one image encoding into another in the most efficient way
    469   available.
   (...)
    491   Yields (label, binary)
    492   """
    493   if isinstance(image_chunks, dict):

File /usr/lib/python3.8/typing.py:813, in _VariadicGenericAlias.__getitem__(self, params)
    811 else:
    812     if not isinstance(args, list):
--> 813         raise TypeError(f"Callable[args, result]: args must be a list."
    814                         f" Got {args}")
    815     params = (tuple(args), result)
    816 return self.__getitem_inner__(params)

TypeError: Callable[args, result]: args must be a list. Got typing.Union[str, int]
```

**Comment by JEElsner** [comment]
This is stemming from another python versioning issue. In Python versions `<3.9`, the code
```python
from typing import Callable
Callable[int,int]
```
doesn't work, because the first argument of Callable should be a list. Beginning in Python `3.9`, Callable automatically turns these situations into a list, that is, it coerces it into `Callable[[int], int]`.

This code is in the `cloudvolume` package. I'll reach out to them about a fix, but the other option is updating the python version of the build environment.

**Comment by JEElsner** [comment]
[Issue in cloud-volume](https://github.com/seung-lab/cloud-volume/issues/622)

**Comment by JEElsner** [comment]
Fixed with cloud-volume version `10.1.0`.
